docs/source/revision_history.rst

```diff
@@ -3,6 +3,12 @@ Revision history
 
 List of major changes:
 
+- Upcoming
```

Comment:
I like this -- it makes it easier to collect everything relevant to the next release in one place (rather than having to scour the logs a day before the release).

Reply:
We do something similar for Pillow: https://github.com/python-pillow/Pillow/blob/master/CHANGES.rst